ludwig/utils/strings_utils.py

```diff
@@ -257,22 +262,29 @@ def _get_sequence_vector(
                 unit_indices_vector[i] = unit_to_id[curr_unit]
             else:
                 unit_indices_vector[i] = unit_to_id[unknown_symbol]
+
+    # Add start and stop symbols.
+    # Huggingface's pretrained tokenizers take care of this implicitly:
+    # https://huggingface.co/docs/transformers/preprocessing
+    if tokenizer_type != "hf_tokenizer":
+        unit_indices_vector = np.append(unit_indices_vector, unit_to_id[STOP_SYMBOL])
+        unit_indices_vector = np.insert(unit_indices_vector, 0, unit_to_id[START_SYMBOL])
     return unit_indices_vector
 
 
 def build_sequence_matrix(
-    sequences,
+    sequences: pd.core.series.Series,
```

Comment:
@tgaddair is this always true? I think it depends on the backend

Reply:
Extended this to `Union[pd.core.series.Series, dd.core.series.Series]`